src/snowflake/connector/telemetry_oob.py

```diff
@@ -377,12 +378,15 @@ def log_http_request_error(
             tags = dict()
         try:
             if self.enabled:
-                telemetry_data = dict()
                 response_status_code = -1
                 # This mimics the output of HttpRequestBase.toString() from JBDC
-                telemetry_data["request"] = f"{method} {url}"
-                telemetry_data["sqlState"] = sqlstate
-                telemetry_data["errorCode"] = errno
+                telemetry_data = generate_telemetry_data(
+                    from_dict={
+                        "request": f"{method} {url}",
+                        "sqlState": sqlstate,
```

Comment:
here's another example of inconsistency: in error.py, method `generate_telemetry_exception_data` uses `KEY_SQLSTATE` as key of which the string value is `SQLState` -- however, here it's `sqlState`